src/plugins/tabs/components/tabs.vue

```diff
@@ -118,6 +125,27 @@ export default {
 
             this.currentTab = tab;
         },
+        showRemoveDialog(tab, index) {
+            if(!this.tabsList[index]) {
```

Comment:
We can just call openmct.overlays.dialog from here, to remove the extra complexity of creating a new RemoveDialog class